src/cluster.cpp

```diff
@@ -421,6 +424,24 @@ CassError cass_cluster_set_use_randomized_contact_points(CassCluster* cluster,
   return CASS_OK;
 }
 
+CassError cass_cluster_set_constant_speculative_execution_policy(CassCluster* cluster,
+                                                                 cass_int64_t constant_delay_ms,
+                                                                 int max_speculative_executions) {
+  if (constant_delay_ms < 0 || max_speculative_executions < 0) {
+    return CASS_ERROR_LIB_BAD_PARAMS;
+  }
+  cluster->config().set_speculative_execution_policy(
+        new cass::ConstantSpeculativeExecutionPolicy(constant_delay_ms,
+                                                     max_speculative_executions));
+  return CASS_OK;
+}
+
+CassError cass_cluster_set_no_speculative_execution_policy(CassCluster* cluster) {
+  cluster->config().set_speculative_execution_policy(
+        new cass::NoSpeculativeExecutionPolicy());
```

Comment:
In general, the driver avoids allocating memory during static initialization (Actually, we try to avoid any sort of complex static initialization) The main reason being that it prevents up from using a custom allocator to make those allocations when the default allocator is overridden.  We've had several users ask for this feature: https://datastax-oss.atlassian.net/browse/CPP-360